core/domain/state_domain.py

```diff
@@ -1339,7 +1349,16 @@ def validate(self):
 
     @classmethod
     def create_default_subtitled_html(cls, content_id):
-        """Create a default SubtitledHtml domain object."""
+        """Create a default SubtitledHtml domain object.
+
+        Args:
+            content_id: str. the ID of the content
+
+        Returns:
+            SubtitledHtml. A default SubtitledHtml domain object. Note that
+            this calls init with html=''.
```

Comment:
Here we want to communicate the properties of the returned object, so we don't need to explain how that object is created. Specifically, instead of explaining that the returned object is created by calling `init` with `html=''`, it might be clearer to say that some attribute of the `SubtitledHtml` object will be `''`